### PR TITLE
Create EIR workshop type

### PR DESCRIPTION
**Pull request by dmcavoy**
Adds a new "course" for EIR workshops called "Admin/Counselor Workshop" and the subject for the workshop is "Welcome".

The workshop is:
- Not Funded
- Suppresses emails
- Does not send exit surveys
- Is not on the map

Screen Shot 2021-09-29 at 2 57 24 PM

## Links

Jira: https://codedotorg.atlassian.net/browse/PLAT-1274

## Testing story

- I added new unit tests for the work on the backend but the workshop_form.jsx compontent on the front end is currently almost entirely untested and setting up tests for it would be significant work. I have logged that we should do some work in this area https://codedotorg.atlassian.net/browse/PLAT-1329

### CODE REVIEW

File: dashboard/app/models/pd/enrollment.rb [hunk 1]
```diff
@@ -175,7 +175,7 @@ def self.filter_for_survey_completion(enrollments, select_completed = true)
 
     # Admin and Counselor still use Pegasus form
     pegasus_enrollments, _ = other_enrollments.partition do |enrollment|
```
Comment (dmcavoy) [inline dashboard/app/models/pd/enrollment.rb]: don't want this to use pegasus enrollment

**Review by tim-dot-org — APPROVED** [review]
looks great to me! were you able to test creating/joining a workshop locally? i'm trying to get it running, but my dev env is a little broken

**Comment by dmcavoy** [comment]
I tried creating it but forgot about joining it. I will do that now before I ship it

**Comment by dmcavoy** [comment]
Seems to be working. Going to ship it!

Screen Shot 2021-09-29 at 9 04 45 PM

**Pull request merged**